projects/kubernetes/notebook.py

```diff
@@ -490,6 +489,58 @@ def get_file_from_pod(filepath):
     return file_content
 
 
+def get_files_from_task(task_name):
+    """
+    Get all files inside a task folder in a notebook server.
+
+    Parameters
+    ----------
+     task_name: str
+
+    Returns
+    -------
+    str
+        File content.
+    """
+
+    warnings.warn(f"Zipping contents of task: '{task_name}'")
+    load_kube_config()
+    api_instance = client.CoreV1Api()
+
+    python_script = (
+        f"import os; "
+        f"os.chdir('{JUPYTER_WORKSPACE}/{task_name}'); "
+        f"os.system('zip -q -r - * | base64'); "
```

Comment:
@andreluiz27 seria bom verificar se um task_name `../` faz exportar o conteúdo da pasta `tasks` (injection).

Reply:
A task passa por uma query: task = self.session.query(models.Task).get(task_id), então caso aconteça um intento de realizar um injection, o trecho: if task is None: raise NOT_FOUND irá reagir com um exception. Testei no postman, mas caso queira posso implementar uma lógica mais precavida